css/ast/src/lib.rs

```diff
@@ -10,7 +10,23 @@ pub struct Text {
 #[ast_node]
 pub struct Stylesheet {
     pub span: Span,
```

Comment:
@kdy1 Does swc usually keep span for all the struct?

Reply:
Yes, many operations related to comment or a source map depend on the span.